pom.xml

```diff
@@ -72,6 +72,8 @@
     <version.shrinkwrap.resolver>2.2.6</version.shrinkwrap.resolver>
     <version.jbossas_spec>3.0.1.Final</version.jbossas_spec>
     <version.dbunit>2.5.3</version.dbunit>
+    <version.snakeyaml>1.18</version.snakeyaml>
+    <version.jackson>1.9.11</version.jackson>
```

Comment:
That was my idea, the problem is that `minimal-json` implements JSON parsing not in a Map style but in a different way. This means that we need to transform its `JsonObject` to `Map` (because it is required by the contract) so it is not a trivial work. So we can leave this for now, or spend some time on migrating it. But definitely not as easy as I thought.